common/attributes/cfme.adoc

```diff
@@ -0,0 +1,6 @@
+
+:product-title: Red Hat CloudForms
```

Comment:
`product-title` is a reserved parameter. AsciiBinder sets this for you based on the `name` parameter in your `_distro_map.yml` file (see this [example](https://github.com/openshift/openshift-docs/blob/master/_distro_map.yml#L3))